components/docs-chef-io/content/automate/backup.md

```diff
@@ -16,7 +16,11 @@ draft = false
 Backups are crucial for protecting your data from catastrophic loss and preparing a recovery procedure.
 The `chef-automate backup create` command creates a single backup that contains data for all products deployed with Chef Automate, including [Chef Infra Server]({{< ref "infra_server.md" >}}) and [Chef Habitat Builder on-prem]({{< ref "on_prem_builder.md" >}}).
 By default, Chef Automate stores backups to the filesystem in the directory `/var/opt/chef-automate/backups`.
-You can also configure Chef Automate to store backups in AWS S3 buckets.
+You can also configure Chef Automate to store backups in AWS S3 buckets or in Google Cloud
+Storage buckets.
+
+See the [restore]{{(< ref "restore.md" >)}} page to learn how to restore a Chef Automate
```

Comment:
Originally these links were in the sections on setting up config, etc., for the various backups. I moved them to the intro where they're readily visible for people who are skimming or who landed on the backup page via a search.